SAMLPortal/wwwroot/js/site.js

```diff
@@ -2,3 +2,47 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+function getResultsFromFilters()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (4)](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)